README.md

```diff
@@ -24,7 +24,8 @@ to compile anywhere, and this is as good a way as any to try and learn it.
 Rust Version
 ------------
 
-uutils supports and is tested on version `1.20.0` and above (including beta and nightly builds) of `rustc`.
+uutils follows Rust's release channels and is tested against stable, beta and nightly.
+To check which one is the latest stable version of Rust visit [Rust's releases page](https://github.com/rust-lang/rust/releases).
```

Comment:
Can you also mention that the oldest supported stable version is `1.22.0` (although `1.20.0` actually works as well other than the tests being broken)?

Reply:
Hmm, I can but then it would have to be updated to the latest stable version every 6 weeks. That's why I added the link. I would have preferred to be able to actually show Rust's current stable version somehow in the README but I don't know how...

Reply:
Ah, only saw now that you already merged and added the remark yourself. :slightly_smiling_face: 
I would still recommend against mentioning the version because of the aforementioned concerns but it's up to you to decide, of course.